packages/components/src/RatioBar/RatioBarComposition.component.js

```diff
@@ -26,6 +34,10 @@ export function RatioBarLine({ percentage, tooltipLabel, className, value }) {
 			style={{
 				flexBasis: `${Math.max(percentage, minPercentage)}%`,
 			}}
+			role={onClick && 'button'}
```

Comment:
you need to handle Enter&space keys to be a11y compliant (yes, I know, not everyone is doing it)

Reply:
with execution onClick function at keydown enter and space keys ?

Reply:
yes: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

`Note: If using role="button" instead of the semantic <button> or <input type="button"> elements, you will need to make the element focusable and have to define event handlers for click and keydown events, including the Enter and Space keys, in order to process the user's input. See the official WAI-ARIA example code.`

Reply:
it's done, but i d'ont know  how to use "aria-pressed", i put at false all time ?

Reply:
you don't have to use aria-pressed as you are not creating a toggle button

Reply:
ok, so, all is good, no ?